go.mod

```diff
@@ -1,3 +1,11 @@
 module github.com/nextmv-io/sdk
 
 go 1.19
+
+require (
+	github.com/google/uuid v1.3.0
+	github.com/gorilla/schema v1.2.0
```

Comment:
https://github.com/gorilla/schema is not maintained anymore

Reply:
We may want to fork it and maintain it ourselves in case we want to still use the dependency

Reply:
As discussed, we accept that this is an unmaintained repo. If we encounter issues, we fork it and fix them.